QUEST_LV_0100.tsv

```diff
@@ -6403,101 +6403,101 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
-QUEST_LV_0100_20150323_006470	{memo X}Chaparition of Escanciu Village took a lot of lives. {nl}Please defeat that monster in the name of the Goddess.
+QUEST_LV_0100_20150323_006465	$Is the antidote ready yet? {nl}The situation's grave. He is gradually getting worse.
+QUEST_LV_0100_20150323_006466	$Evoniphon sure employed a vicious poison. {nl}We need to hunt him down before he causes more damage... 
+QUEST_LV_0100_20150323_006467	$Oh, is this the antidote? {nl}Thank you very much. I will let him take it right away. {nl}
+QUEST_LV_0100_20150323_006468	$If it wasn't for you, many workers could have lost their lives for nothing. {nl}I will report about this to Sir Uska. {nl}Truly... You were sent by the Goddess!{nl}
+QUEST_LV_0100_20150323_006469	$The monsters in the Owl Burial Ground should be a fair bet for you. {nl}Right? 
+QUEST_LV_0100_20150323_006470	{memo X}The Chaparition in Escanciu Village took a lot of lives. {nl}Please defeat that monster in the name of the Goddess.
 QUEST_LV_0100_20150323_006471	{memo X}I'm sorry but can you burn this oration in front of the epitaph in Rukas Plateau? {nl}You have explored different areas of the valley. {nl}I think you are more than qualified to comfort their souls. 
-QUEST_LV_0100_20150323_006472	It's going for a killing spree of the monsters in the Owl Grave. {nl}Around 40 of them? I bet that you can't make it.
-QUEST_LV_0100_20150323_006473	I'll go to Ramstis Ridge and release the seal
-QUEST_LV_0100_20150323_006474	Let's go help Grita in Fedimian Outskirts
+QUEST_LV_0100_20150323_006472	$Try and perform a killing spree of the monsters in the Owl Burial Ground. {nl}Around 40 of them? I bet that you can't make it.
+QUEST_LV_0100_20150323_006473	$I'll go to Ramstis Ridge and release the seals
+QUEST_LV_0100_20150323_006474	$Let's go help Grita in Fedimian Outskirts
 QUEST_LV_0100_20150323_006475	{memo X}
-QUEST_LV_0100_20150323_006476	About the barrier of Nefritas Cliff
-QUEST_LV_0100_20150323_006477	Owl Grave Test
-QUEST_LV_0100_20150323_006478	Tyrant of the Srautas Gorge
-QUEST_LV_0100_20150323_006479	{memo X}Notice/Find the elderly of Vieta Gorge!#5
-QUEST_LV_0100_20150323_006480	Curse of Laziness - Attack of Soul
+QUEST_LV_0100_20150323_006476	$About the barriers in Nefritas Cliff
+QUEST_LV_0100_20150323_006477	$Owl Burial Ground Test
+QUEST_LV_0100_20150323_006478	$Tyrant of the Srautas Gorge
+QUEST_LV_0100_20150323_006479	{memo X}Notice/Find the elderly in Vieta Gorge!#5
+QUEST_LV_0100_20150323_006480	$Curse of Laziness - Spirit Attack
 QUEST_LV_0100_20150323_006481	{memo X}Notice/!/A piece of paper fell#3
 QUEST_LV_0100_20150323_006482	{memo X}Notice/Mutated Tree Root Crystal is disappeared!#5
-QUEST_LV_0100_20150323_006483	Give the portrait and handkerchief
-QUEST_LV_0100_20150323_006484	It's scary. Run away
-QUEST_LV_0100_20150323_006485	Leave
+QUEST_LV_0100_20150323_006483	$Give the portrait and handkerchief
+QUEST_LV_0100_20150323_006484	$It's frightening. Make a run from it!
```

Comment:
"Make a run from it." doesn't really sound like something someone would say naturally. "Run away!", "Run for it!" or "Make a run for it!" would be better.